rails_application/app/views/orders/show.html.erb

```diff
@@ -23,8 +23,8 @@
     <%= button_to("Pay", pay_order_path(@order.uid), class: "mr-3 ml-3 inline-flex items-center px-4 py-2 border rounded-md shadow-sm text-sm font-medium focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-blue-500 border-transparent text-white bg-blue-600 hover:bg-blue-700") %>
   <% end %>
 
-  <% if @order.state == "Submitted" %>
-    <%= link_to("Refund", new_order_refund_path(order_id: @order.uid), class: "mr-3 inline-flex items-center px-4 py-2 border rounded-md shadow-sm text-sm font-medium focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-blue-50 border-gray-300 text-gray-700 bg-white hover:bg-gray-50") %>
+  <% if @order.state == "Paid" %>
```

Comment:
To make reviewing process simpler and smoother this change should be a separate commit :) Just for the future